create_conversation.py

```
@@ -0,0 +1,167 @@
+            details = details_response.json()
+
+            print(f'   Title: {details.get("title", "N/A")}')
+            print(f'   Status: {details.get("status", "N/A")}')
+            print(f'   LLM Model: {llm_model or "Not detected yet"}')
+            print(f'   Created At: {details.get("created_at", "N/A")}')
+            print(f'   Repository: {details.get("repository", "N/A")}')
+
+            # Print runtime details if available
+            if 'url' in details:
+                print(f'   Runtime URL: {details["url"]}')
+            if 'session_api_key' in details and details['session_api_key']:
```

Ruff (0.13.1)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)